{"uuid": "857517e3-a29c-4d6f-8cd2-64a09620cfb0", "children": ["ea281893-b4b5-4a10-871b-af2b66ab6073"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/e2e/basic/conftest.py\", line 80, in setup_profiles\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/fixtures_4x.py\", line 111, in setup_profiles\n    instantiate_profile_obj.check_and_set_ssid_sec(\"2G\", profile_data[\"ssid\"][mode][0][\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 677, in check_and_set_ssid_sec\n    self.get_all_ssid_detail(radio=radio)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 724, in get_all_ssid_detail\n    ssid_details = self.run_generic_cli_command(cmd)\n  File \"/home/imgd/Projects/ctpriv-scripts/altice/wlan-testing/tests/../libs/controller/controller_4x/controller.py\", line 180, in run_generic_cli_command\n    output = os.popen(cmd).read()\n"}, "start": 1702366990322, "stop": 1702370234597}], "start": 1702366990322, "stop": 1702370235006}